manifests/init.pp

```diff
@@ -96,23 +96,34 @@
 #     An array of API endpoints to connect uchiwa to one or multiple sensu servers.
 #
 class uchiwa (
-  $package_name         = $uchiwa::params::package_name,
-  $service_name         = $uchiwa::params::service_name,
-  $version              = $uchiwa::params::version,
-  $install_repo         = $uchiwa::params::install_repo,
-  $repo                 = $uchiwa::params::repo,
-  $repo_source          = $uchiwa::params::repo_source,
-  $repo_key_id          = $uchiwa::params::repo_key_id,
-  $repo_key_source      = $uchiwa::params::repo_key_source,
-  $manage_services      = $uchiwa::params::manage_services,
-  $manage_user          = $uchiwa::params::manage_user,
-  $host                 = $uchiwa::params::host,
-  $port                 = $uchiwa::params::port,
-  $user                 = $uchiwa::params::user,
-  $pass                 = $uchiwa::params::pass,
-  $refresh              = $uchiwa::params::refresh,
-  $sensu_api_endpoints  = $uchiwa::params::sensu_api_endpoints,
-) inherits uchiwa::params {
+  $package_name         = 'uchiwa',
+  $service_name         = 'uchiwa',
+  $version              = 'latest',
+  $install_repo         = true,
+  $repo                 = 'main',
+  $repo_source          = undef,
+  $repo_key_id          = '7580C77F',
+  $repo_key_source      = 'http://repos.sensuapp.org/apt/pubkey.gpg',
+  $manage_services      = true,
+  $manage_user          = true,
+  $host                 = '0.0.0.0',
+  $port                 = '3000',
+  $user                 = '',
+  $pass                 = '',
+  $users                = [],
```

Comment:
i'm not sure i understand what you mean - but it is in line with the
puppet-sensu module, and it's got the docs above as well regardless...

On Thu, Jul 9, 2015 at 11:19 PM, Kyle Anderson <notifications@github.com>
wrote:

> In manifests/init.pp
> <https://github.com/Yelp/puppet-uchiwa/pull/47#discussion_r34325269>:
>
> > -) inherits uchiwa::params {
> > +  $package_name         = 'uchiwa',
> > +  $service_name         = 'uchiwa',
> > +  $version              = 'latest',
> > +  $install_repo         = true,
> > +  $repo                 = 'main',
> > +  $repo_source          = undef,
> > +  $repo_key_id          = '7580C77F',
> > +  $repo_key_source      = 'http://repos.sensuapp.org/apt/pubkey.gpg',
> > +  $manage_services      = true,
> > +  $manage_user          = true,
> > +  $host                 = '0.0.0.0',
> > +  $port                 = '3000',
> > +  $user                 = '',
> > +  $pass                 = '',
> > +  $users                = [],
>
> Yea... but looking at this now I don't know what it is in reference to.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Yelp/puppet-uchiwa/pull/47/files#r34325269>.
>